x-pack/plugins/observability_shared/public/components/feature_feedback_button/feature_feedback_button.tsx

```diff
@@ -25,24 +25,40 @@ const getDeploymentType = (isCloudEnv?: boolean, isServerlessEnv?: boolean): str
 
 const getSurveyFeedbackURL = (
   formUrl: string,
+  formConfig?: FormConfig,
   kibanaVersion?: string,
   deploymentType?: string,
   sanitizedPath?: string
```

Comment:
Maybe we could refactor these args to be an object instead?
```suggestion
  {
  formUrl: string,
  formConfig?: FormConfig,
  kibanaVersion?: string,
  deploymentType?: string,
  sanitizedPath?: string
  }
```

Reply:
Good suggestion, thanks! Done ✅